Sources/Helpers/OAAppSettings.h

```diff
@@ -679,6 +679,8 @@ typedef NS_ENUM(NSInteger, EOARateUsState)
 @property (nonatomic) OAApplicationMode* applicationMode;
 @property (nonatomic) OACommonString* availableApplicationModes;
 @property (nonatomic) OACommonAppMode* defaultApplicationMode;
+@property (nonatomic) OACommonAppMode* carPlayMode;
+@property (nonatomic) OACommonBoolean *isCarPlayModeDefault;
```

Comment:
I'm sure this can be done with only one setting.

Reply:
Don't do this